src/components/victory-area/helper-methods.js

```diff
@@ -6,6 +6,13 @@ import Scale from "../../helpers/scale";
 
 export default {
 
+  getScale(props, fallbackProps) {
+    const modifiedProps = Helpers.modifyProps(props, fallbackProps);
+    const {scale} = this.getCalculatedValues(modifiedProps);
```

Comment:
fixed https://github.com/FormidableLabs/victory-chart/pull/333/commits/87ac4a273fdebcd630261e51fa624c10b65b2ac7 , and https://github.com/FormidableLabs/victory-chart/pull/333/commits/e904b1f498ce98099e44a70b2edf58b6169c066e